src/transformStateWithClones.js

```diff
@@ -8,6 +8,38 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  let clone = {};
+
+  for (let i = 0; i < actions.length; i++) {
+    if (result.length === 0) {
+      clone = Object.assign({}, state);
+    } else {
+      clone = Object.assign({}, result[result.length - 1]);
+    }
```

Comment:
With clone being created above you dont need this anymore. Just add new properties or remove existing ones when needed
```suggestion
```